server/src/migrations/Migration20210406120019.ts

```diff
@@ -0,0 +1,21 @@
+import { Migration } from '@mikro-orm/migrations';
```

Comment:
Please do not include migration files. If somebody else's database looks different from yours, this code won't work. Everybody should run npm run create:migration to create their migrations.

Reply:
ok ok